src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/callback/CreateCaseValidator.java

```diff
@@ -66,18 +73,56 @@ public CreateCaseValidator() {
     }
 
     public Validation<Seq<String>, ExceptionRecord> getValidation(CaseDetails caseDetails) {
-        return Validation
-            .combine(
-                hasCaseTypeId(caseDetails),
-                hasPoBox(caseDetails),
-                hasJurisdiction(caseDetails),
-                hasJourneyClassification(caseDetails),
-                hasDateField(caseDetails, "deliveryDate"),
-                hasDateField(caseDetails, "openingDate"),
-                getScannedDocuments(caseDetails),
-                getOcrDataFields(caseDetails)
-            )
-            .ap(ExceptionRecord::new);
+
+        Validation<String, String> hasCaseTypeId = hasCaseTypeId(caseDetails);
+        Validation<String, String> hasPoBox = hasPoBox(caseDetails);
+        Validation<String, String> hasJurisdiction = hasJurisdiction(caseDetails);
+        Validation<String, String> hasFormType = hasFormType(caseDetails);
+        Validation<String, Classification> hasJourneyClassification = hasJourneyClassification(caseDetails);
+        Validation<String, Instant> hasDeliveryDate = hasDateField(caseDetails, "deliveryDate");
+        Validation<String, Instant> hasOpeningDate = hasDateField(caseDetails, "openingDate");
+        Validation<String, List<ScannedDocument>> hasScannedDocuments = getScannedDocuments(caseDetails);
+        Validation<String, List<OcrDataField>> hasOcrDataFields = getOcrDataFields(caseDetails);
+
+        List<Validation<String, ?>> validations = Arrays.asList(
+            hasCaseTypeId,
+            hasPoBox,
+            hasJurisdiction,
+            hasFormType,
+            hasJourneyClassification,
+            hasDeliveryDate,
+            hasOpeningDate,
+            hasScannedDocuments,
+            hasOcrDataFields
+        );
+
+        List<String> errors = getValidationErrors(validations);
+        ExceptionRecord exceptionRecord = null;
+        if (CollectionUtils.isEmpty(errors)) {
+            exceptionRecord = new ExceptionRecord(
+                hasCaseTypeId.get(),
+                hasPoBox.get(),
+                hasJurisdiction.get(),
+                hasJourneyClassification.get(),
+                hasFormType.get(),
+                hasDeliveryDate.get(),
+                hasOpeningDate.get(),
+                hasScannedDocuments.get(),
+                hasOcrDataFields.get()
+            );
+        }
+
+        return isNotEmpty(errors)
+            ? Validation.fromEither(Either.left(io.vavr.collection.List.ofAll(errors)))
+            : Validation.fromEither(Either.right(exceptionRecord));
+    }
```

Comment:
how about just returning `Validation.valid(...)` inside the if statement and in the else part - `return Validation.invalid(errors)`